packages/scan/src/Scan.spec.ts

```diff
@@ -304,6 +304,59 @@ describe('Scan', () => {
     });
   });
 
+  describe('expect with failFast: false', () => {
```

Comment:
Please follow the conventions. All of these test cases should be moved under `describe('expect')` suite with a proper titles